### PR TITLE
[mcs] Fix platform-specific build of "build" profile and remove more remnants of the "basic" profile

### PR DESCRIPTION
**Pull request by akoeplinger**
The platform specific build got lost in 8d8fd97dbfc6f4213e0fe9fca7574808dd8496d3 but we need it for monolite. The remnants of the "basic" profile are no longer needed.

### CODE REVIEW

**Comment by akoeplinger** [comment]
@monojenkins build deb

**Comment by akoeplinger** [comment]
@monojenkins build deb

File: mcs/build/Makefile [hunk 1]
```diff
@@ -52,7 +51,6 @@ DISTFILES = \
 	corcompare.make			\
 	corcompare-api.xsl		\
 	executable.make			\
-	gensources.cs			\
```
Comment (luhenry) [inline mcs/build/Makefile]: Looks like a typo.

Comment (akoeplinger) [reply mcs/build/Makefile]: No, this file was moved to somewhere else in 8d8fd97.

**Pull request merged**